src/NovaEditorJsConverter.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+namespace Advoor\NovaEditorJs;
+
+use Closure;
+use EditorJS\EditorJS;
+use EditorJS\EditorJSException;
+use Illuminate\Support\HtmlString;
+use JsonException;
+
+class NovaEditorJsConverter
+{
+    /**
+     * List of callbacks that can render blocks
+     * @var array<Closure>
+     */
+    protected array $renderCallbacks = [];
+
+    public function __construct()
+    {
+        $this->registerDefaultCallbacks();
+    }
+
+    /**
+     * Add a custom render callback for the given block.
+     *
+     * @param string   $block  Name of the block, as defined in the JSON
+     * @param callable $callback  Closure that returns a string (or a Stringable)
+     * @return void
+     */
+    public function addRender(string $block, callable $callback): void
+    {
+        $this->renderCallbacks[$block] = $callback;
+    }
+
+    /**
+     * Renders the given EditorJS data to safe HTML.
+     *
+     * @param iterable|object|string $data Data to render
+     * @return HtmlString           Safe, directly returnable string.
+     */
+    public function generateHtmlOutput(iterable|object|string $data): HtmlString
```

Comment:
Iterable is an object type, which makes it redundant

```suggestion
    public function generateHtmlOutput(object|string $data): HtmlString
```